lib/internal/streams/from.js

```diff
@@ -25,18 +26,23 @@ function from(Readable, iterable, opts) {
     });
   }
 
-  if (iterable && iterable[SymbolAsyncIterator])
+  let isAsync = false;
+  if (iterable && iterable[SymbolAsyncIterator]) {
+    isAsync = true;
     iterator = iterable[SymbolAsyncIterator]();
-  else if (iterable && iterable[SymbolIterator])
+  } else if (iterable && iterable[SymbolIterator]) {
+    isAsync = false;
     iterator = iterable[SymbolIterator]();
-  else
+  } else {
     throw new ERR_INVALID_ARG_TYPE('iterable', ['Iterable'], iterable);
+  }
 
   const readable = new Readable({
     objectMode: true,
     highWaterMark: 1,
+    // me
```

Comment:
What does this mean?

Reply:
Personal mark for "may be I can resolve this" ;)
Forget to remove in PR, sorry

Reply:
fixed